tests/integration/tier0/inhibit-if-kmod-is-not-supported/main.fmf

```diff
@@ -0,0 +1,39 @@
+summary: Unsupported kmod behavior
+description: |
+    Load kernel module that is not supported in RHEL and verify the conversion is inhibited.
+    Verify, that removing this kmod does not interfere with new conversion run.
+    Verify that loading custom kernel module, that marks the kernel as "tainted", inhibits the conversion.
+
+tier: 0
+
+tag+:
+    - kmod
+    - kernel
+    - inhibit-if-kmod-is-not-supported
+
+/custom_module_loaded:
+    tag+:
+        - custom-module-loaded
+    test: |
+        pytest -svv -m custom_module_loaded
+
+/custom_module_not_loaded:
+    tag+:
+        - custom-module-not-loaded
+    test: |
+        pytest -svv -m custom_module_not_loaded
+
+/force_loaded_kmod:
+    adjust+:
+        - enabled: false
+          when: distro == centos-7 or distro == oraclelinux-7
```

Comment:
I would add a comment mentioning why we disable this test for 7 distros. Something like that force-loading kmod on those distros doesn't always work or is behaving inconsistently

Reply:
Good point, I will add the comment.